core/Twig.php

```diff
@@ -199,32 +227,32 @@ function ($value) {
         $this->twig->addTest($test);
     }
 
-    protected function addFunctionGetJavascriptTranslations()
+    protected function addFunctionGetJavascriptTranslations(): void
     {
         $getJavascriptTranslations = new TwigFunction(
             'getJavascriptTranslations',
-            array(StaticContainer::get('Piwik\Translation\Translator'), 'getJavascriptTranslations')
+            [StaticContainer::get(Translator::class), 'getJavascriptTranslations']
         );
         $this->twig->addFunction($getJavascriptTranslations);
     }
 
-    protected function addFunctionIsPluginLoaded()
+    protected function addFunctionIsPluginLoaded(): void
     {
         $isPluginLoadedFunction = new TwigFunction('isPluginLoaded', function ($pluginName) {
             return \Piwik\Plugin\Manager::getInstance()->isPluginLoaded($pluginName);
         });
         $this->twig->addFunction($isPluginLoadedFunction);
     }
 
-    protected function addFunctionIncludeAssets()
+    protected function addFunctionIncludeAssets(): void
     {
         $includeAssetsFunction = new TwigFunction('includeAssets', function ($params) {
             if (!isset($params['type'])) {
                 throw new Exception("The function includeAssets needs a 'type' parameter.");
             }
 
             $assetType = strtolower($params['type']);
-            $deferJs = boolval($params['defer'] ?? false);
+            $deferJs           = boolval($params['defer'] ?? false);
```

Comment:
Is this space a mistake?

Reply:
Of, I see, it’s a tab instead of a space.

```suggestion
            $deferJs   = boolval($params['defer'] ?? false);
```